tests/helpers/test_entity.py

```diff
@@ -927,5 +927,8 @@ def test_entity_category_schema(value, expected):
 def test_entity_category_schema_error(value):
     """Test entity category schema."""
     schema = vol.Schema(entity.ENTITY_CATEGORIES_SCHEMA)
-    with pytest.raises(vol.Invalid):
+    with pytest.raises(
+        vol.Invalid,
+        match=r"expected EntityCategory or one of 'config', 'diagnostic', 'system'",
```

Comment:
Home Assistant would previously raise
```
    value must be one of ['config', 'diagnostic', 'system']
```

Reply:
There is a match keyword ?! that's so much easier than what I normally do.